backend/src/middleware/klicktippMiddleware.ts

```diff
@@ -31,15 +30,13 @@ export const klicktippNewsletterStateMiddleware: MiddlewareFn = async (
   // eslint-disable-next-line n/callback-return
   const result = await next()
   let klickTipp = new KlickTipp({ status: 'Unsubscribed' })
-  if (CONFIG.KLICKTIPP) {
-    try {
-      const klickTippUser = await getKlickTippUser(result.email)
-      if (klickTippUser) {
-        klickTipp = new KlickTipp(klickTippUser)
-      }
-    } catch (err) {
-      logger.error(`There is no user for (email='${result.email}') ${err}`)
+  try {
+    const klickTippUser = await getKlickTippUser(result.email)
+    if (klickTippUser) {
+      klickTipp = new KlickTipp(klickTippUser)
```

Comment:
I'd like to discuss if we really need this middleware - what are downside, what are benefits?